cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py

```diff
@@ -58,16 +58,29 @@ class CxTowerCommandExecuteWizard(models.TransientModel):
     show_servers = fields.Boolean(
         compute="_compute_show_servers",
     )
+    invisible_run_wizard = fields.Boolean(compute="_compute_invisible_run_wizard")
```

Comment:
Why do we need an extra field here? This should be done with a domain on the form.

Reply:
Thanks, understood
I'll fix it